Makefile

```diff
@@ -43,14 +43,38 @@ test: build
 	$(GOTEST) -v ./... 2>&1
 
 integration-test: build
+	mkdir -p dist
 	# temporary workaround, remove next line when x/net false positive is fixed
 	echo 'CVE-2018-17142 until=2020-10-22 #x/net false positive\nCVE-2018-17846 until=2020-10-22\nCVE-2018-17143 until=2020-10-22\nCVE-2018-17847 until=2020-10-22\nCVE-2018-17848 until=2020-10-22' > packages/testdata/.nancy-ignore
 	cd packages/testdata && GOPATH=. ../../$(BINARY_NAME) sleuth -p Gopkg.lock && cd -
 	# temporary workaround, remove next line when x/net false positive is fixed
 	mv packages/testdata/.nancy-ignore .
 	go list -json -m all | ./$(BINARY_NAME) sleuth
 	go list -m all | ./$(BINARY_NAME) sleuth
-	go list -json -m all > deps.out && ./$(BINARY_NAME) sleuth < deps.out
-	go list -m all > deps.out && ./$(BINARY_NAME) sleuth < deps.out
+	go list -json -m all > dist/deps.out && ./$(BINARY_NAME) sleuth < dist/deps.out
+	go list -m all > dist/deps.out && ./$(BINARY_NAME) sleuth < dist/deps.out
 	# temporary workaround, remove next line when x/net false positive is fixed
 	rm .nancy-ignore
+
+build-linux:
+	GOOS=linux GOARCH=amd64 $(GO_BUILD_FLAGS) $(GOBUILD) -o $(BINARY_NAME) -v
+
+docker-alpine-integration-test: build-linux
+	mkdir -p dist
+	docker build . -f Dockerfile.alpine -t sonatypecommunity/nancy:alpine-integration-test
+	# create file, volume mount to simulate, ci run of the container and things just happening inside the container instead of passing output to the container directly
+	go list -json -m all > dist/deps.out
+	echo "cat /tmp/dist/deps.out | nancy sleuth" > dist/ci.sh
+	chmod +x dist/ci.sh
+	# run the container....using cat with no params keeps it running
+	docker run --name alpine-integration-test -td sonatypecommunity/nancy:alpine-integration-test cat
+	# copy the code as if it was actually in the "ci" container.. doing this cause circleci cant actually mount volumes
+	docker cp . alpine-integration-test:/tmp
```

Comment:
A question here, is why not in `Dockerfile.alpine` do a `COPY dist dist` or something akin, and control it via a build arg? Was it just not wanting to modify the Dockerfile itself?

Reply:
I'm not suggesting that approach, just you know, unpacking this approach.

Reply:
So yah didnt want to make the docker file bigger and package the dist directory in it. This is just an attempt to simulate as if you did the following. 

```
build:
    docker:
      - image: sonatypecommunity/nancy:alpine
    working_directory: /go/src/github.com/sonatype-nexus-community/nancy
    steps:
       - nancy (sleuth|iq) ...
```